Source/Extensions/Blazorise.Charts/LambdaConverter.cs

```diff
@@ -0,0 +1,139 @@
+﻿#region Using directives
+using System;
+using System.Linq.Expressions;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Lambda2Js;
+#endregion
+
+namespace Blazorise.Charts
+{
+    public class LambdaConverter<T> : JsonConverter<Expression<T>>
+    {
+        public override Expression<T> Read( ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options )
+        {
+            throw new NotImplementedException();
+        }
+
+        public override void Write( Utf8JsonWriter writer, Expression<T> value, JsonSerializerOptions options )
+        {
+            var jsBody = value.CompileToJavascript( new JavascriptCompilationOptions( JsCompilationFlags.BodyOnly ) );
+
+            var result = BuildFunction( value, jsBody );
+
+            writer.WriteStringValue( result );
+        }
+
+        static string BuildFunction( Expression<T> expression, string jsBody )
+        {
+            var sb = new StringBuilder();
+
+            sb.Append( "function(" );
+            sb.Append( BuildFunctionParameters( expression ) );
+            sb.Append( ") {" );
+
+            sb.Append( "return " );
+            sb.Append( jsBody );
+            sb.Append( "; }" );
+
+            return sb.ToString();
+        }
+
+        static string BuildFunctionParameters( Expression<T> value )
```

Comment:
Should this just take in the existing SB instead of making a new one?

Reply:
Yeah it can.